src/shared/utils/utils.test.ts

```diff
@@ -225,4 +225,42 @@ describe("Util Functions", () => {
       expect(utils.getIssuerAddress(document)).toStrictEqual("0xabcf");
     });
   });
+
+  describe("getMerkleRoot", () => {
```

Comment:
It should throw if not v2 or v3

Reply:
Cause currently the v2 check is not strict thus even if the document is not v2 schema, it will still pass the v2 test. I placed the default throw error is for the future when the checking of v2 schema has changed.